data/json/items/comestibles/mutagen.json

```diff
@@ -786,4 +835,4 @@
     "fun": -20,
     "vitamins": [ [ "mutagen", 60 ] ]
   }
-]
+]
```

Comment:
Not surr what got changed there but it's better to not introduce unecessary changes

Reply:
It's a space that was removed when I linted the file.
I'll add it back

Reply:
Done